x/payment/keeper/storage_fee_charge.go

```diff
@@ -126,15 +126,15 @@ func (k Keeper) applyFrozenUserFlows(ctx sdk.Context, userFlows types.UserFlows,
 	//var frozenRateChanges []types.StreamRecordChange
 	totalActiveRate, totalFrozenRate := sdk.ZeroInt(), sdk.ZeroInt()
 	for _, flowChange := range userFlows.Flows {
-		outFlow := k.GetOutFlow(ctx, sdk.MustAccAddressFromHex(streamRecord.Account), types.OUT_FLOW_STATUS_FROZEN, sdk.MustAccAddressFromHex(flowChange.ToAddress))
+		outFlow := k.GetOutFlow(ctx, sdk.MustAccAddressFromHex(streamRecord.Account), types.OUT_FLOW_STATUS_ACTIVE, sdk.MustAccAddressFromHex(flowChange.ToAddress))
```

Comment:
Did not quite get this.

Reply:
reject migration when the account is frozen in both start and end tx.

Reply:
the following if/else is also switched. the new implementation can handle adding flows when the account is frozen.